core/include/msg.h

```diff
@@ -375,16 +375,21 @@ int msg_reply_int(msg_t *m, msg_t *reply);
  *
  * @param[in] pid    a PID
  *
- * @return Number of messages available in queue of @p pid on success
- * @return 0, if no caller's message queue is initialized
+ * @return  Number of messages available in queue of the thread identified by
+ *          PID @p pid
+ * @retval  0       The message queue of the thread identified by PID @p pid is
+ *                  *not* initialized
+ * @retval  0       PID @p pid does not refer to a running thread
```

Comment:
```suggestion
 * @retval  0       The message queue of the thread identified by PID @p pid is
 *                  *not* initialized or PID @p pid does not refer to a running thread
```

otherwise CI complains, and it's also maybe clearer that way.